src/utilities/valueFromAST.js

```diff
@@ -57,13 +59,17 @@ export function valueFromAST(
   }
 
   if (!valueAST) {
+    return undefined;
```

Comment:
I learned from `eslint`s [consistent-return](http://eslint.org/docs/rules/consistent-return) rule that explicit is better than explicit.

Second reason is semantically big difference between `null` and `undefined`.

And last, every minifier will replace `return undefined;` with simple `return;`